db/dcrsqlite/dcrsqlite_test.go

```diff
@@ -30,3 +32,42 @@ func InitTestDB(targetDBFile string) *DB {
 	}
 	return db //is not nil
 }
+
+// checkChartsDataIsDefault checks if chartsData is in default mode:
+//  - all the returned arrays expected to be empty
+//  - chartsData is not null (nil)
+//
+// These checks are sorted according to the ChartsData struct fields order
+func checkChartsDataIsDefault(testedFunctionName string, chartsData *dbtypes.ChartsData) {
+	if chartsData == nil {
+		testutil.ReportTestFailed(
+			testedFunctionName + "() failed: value is nil")
+	}
+	// Array checks:
```

Comment:
I don't see the necessity of all these lines.  If you want to make sure that `chartsData` is initialized to the zero values, just do:

```go
var emptyChartsData dbtypes.ChartsData
if !reflect.DeepEqual(emptyChartsData, *chartsData) {
```

Reply:
DeepEqual wouldn't provide detailed info on what went wrong (which field is incorrect).